projects/core/src/lib/core.module.ts

```diff
@@ -1,10 +1,17 @@
+import { AuthorizationModule } from "./authorization/authorization.module";
 import { NgModule } from "@angular/core";
+import { StoreModule } from "@ngrx/store";
 import { CoreComponent } from "./core.component";
-import { AuthorizationModule } from "./authorization/authorization.module";
+import { CommonModule } from "@angular/common";
+import { CommonComponentsModule } from "@jsmu/common-components";
 
 @NgModule({
 	declarations: [CoreComponent],
-	imports: [AuthorizationModule],
-	exports: [CoreComponent]
+	imports: [
+		CommonModule,
+		CommonComponentsModule,
+		AuthorizationModule
+		// StoreModule.forFeature()
```

Comment:
plrase uncomment

Reply:
I can't, because it's require a parameters, and if i don't mistake i'll need create a reducer and etc